.licenses/npm/@actions/http-client-2.1.1.dep.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: "@actions/http-client"
+version: 2.1.1
+type: npm
+summary: Actions Http Client
+homepage: https://github.com/actions/toolkit/tree/main/packages/http-client
+license: other
```

Comment:
Could you please change `other` to `mit`

Reply:
Thanks, done.